packages/shared-base/src/sentry/index.ts

```diff
@@ -0,0 +1,78 @@
+import type { PersonaIdentifier, ProfileIdentifier } from '../Identifier/index.js'
+
+// @ts-ignore
+const Sentry = globalThis.Sentry as Sentry
+
+Sentry.init({
```

Comment:
only init it when user choose to opt-in the logging

Reply:
yep. I will add opt-in in next commit.